### PR TITLE
fix: authorized api requests

### PR DESCRIPTION
**Pull request by guilhermeicarofr**
code fix to interact correctly to back-end. Axios requests now receive config object containing bearer token authorization, retrieved from login context

### CODE REVIEW

File: src/components/pages/Timeline.js [hunk 1]
```diff
@@ -25,7 +28,7 @@ function Timeline() {
 					"An error occured while trying to fetch the posts, please refresh the page"
 				);
 			});
-	}, [refresh]);
+	}, [refresh, config]);
```
Comment (SauloDriven) [inline src/components/pages/Timeline.js:31]: Nice

**Review by SauloDriven — APPROVED** [review]
:100:

File: src/components/pages/Timeline.js (hunk 1 above)
Comment (guilhermeicarofr) [reply src/components/pages/Timeline.js:31]: Thanks!

**Review by Francksuel — APPROVED** [review]
LGTM, i think we have to implement in the new requests.

**Pull request merged**